app/_data/docs_nav_kuma_2.0.x.yml

```diff
@@ -90,28 +91,39 @@ items:
             url: "/quickstart/universal/#next-steps"
       - text: Standalone deployment
         url: /deployments/stand-alone/
+        src: /production/deployment/stand-alone/
         items:
```

Comment:
@fabianrbz In the 2.1 nav, I had to remove all the links to headers from pages that I moved because they resulted in errors when I ran make build. Ex. I had to remove "Failure modes" (src: /production/stand-alone/#failure-modes) for the Standalone deployment page. But I left them in the 2.0 nav as is and they were fine and displayed correctly?

Reply:
@cloudjumpercat did you leave the hash portion of the URL in the `src`?
e.g. `/production/deployment/stand-alone/#standalone-deployment` should be `/production/deployment/stand-alone/`, it should point to the file name not the url

Reply:
@fabianrbz I did. Ah, so if I do:
```
- text: About
            url: "/deployments/stand-alone/#standalone-deployment"
            src: /production/deployment/stand-alone/
```
That would take me to the correct section on that page?

Reply:
yes, `src` just tells the code to render that file for that page, the `hash` portion of the `url` will make the page scroll to the corresponding section